### ISSUE TITLE
port from darkplaces

### ISSUE THREAD

**Issue by EndlessEden** [nadako/HaxeQuake#11]
You should look into porting the darkplaces engine, into this. Significant leaps and bounds have been made in lordhavoc's engine over the ID source.

Not to mention it would add support for newer Q1 mods.

**Comment by nadako** [comment]
Yeah, it is awesome, but it's also HUGE and I'm not sure I'll have time and motivation to port stuff from it. QuakeSpasm is a better target - supporting newer map formats and whatnot and still being close to vanilla Quake.

**Comment by plankatron** [comment]
https://github.com/plankatron/quakemash  this is a very large hmap2 map you can test with :)
https://github.com/plankatron/quakemash/tree/master/release
